code/game/objects/structures/girders.dm

```diff
@@ -222,10 +222,18 @@
 		if(2.0)
 			if (prob(30))
 				dismantle()
-			return
+				return
+			else
+				health -= rand(60,180)
+
 		if(3.0)
 			if (prob(5))
 				dismantle()
-			return
+				return
+			else
+				health -= rand(40,80)
```

Comment:
girder ex_act altered mostly for the sake of hatton, and also for future plans with mechs